rtCommon/readDicom.py

```diff
@@ -105,3 +106,88 @@ def writeDicomToBuffer(dicomImg):
     dataBytesIO.seek(0)
     data = dataBytesIO.read()
     return data
+
+def getSubjectInterpretation(cfg):
```

Comment:
With these added functions I feel that the file name "readDicom.py" is not reflective anymore of the content. Maybe we should rename this file or make a new python file with these image helper functions in it.